pages.pt_BR/linux/adig.md

```diff
@@ -0,0 +1,24 @@
+# adig
+
+> Imprime informações recebidas dos servidores do Sistema de Domínio de Nome(DNS).
+> Mais informações: <https://manned.org/adig>.
+
+- Exibe uma gravação A (padrão) do DNS por nome(s) de servidor(es):
+
+`adig {{examplo.com}}`
+
+- Exibe uma saída de [d]epuração extra:
+
+`adig -d {{examplo.com}}`
+
+- Conecta ao servidor DNS [e]specificado:
+
+`adig -s {{1.2.3.4}} {{examplo.com}}`
```

Comment:
```suggestion
- Conecte-se a um servidor DNS específico:

`adig -s {{1.2.3.4}} {{examplo.com}}`
```

Optional fix (error source: original page): unexpected `specified` term, `specific` expected.
Note: `Connect to a [s]pecific DNS server:` can be used a translation source.